src/main/java/com/enterprise/fastfoodapplication/dto/Order.java

```diff
@@ -2,21 +2,17 @@
 
 import lombok.Data;
 
-public @Data
-class CartOrder {
-
-    public CartOrder(){
-    }
+/*
+Order class holds food and quantity
+cartID - used to create order stub until submitted as order
+orderID - populated when order is created. used for order history queries
+* */
 
-    public CartOrder(Food food){
-        this.food = food;
-        this.foodQuantity = 1;
-    }
-
-    public CartOrder(Food food, int foodQuantity) {
-        this.food = food;
-    }
+public @Data
+class Order {
 
+    private String orderId;
+    private String cartId;
```

Comment:
Are these used anywhere?

This appears to be an incomplete feature.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality .
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.